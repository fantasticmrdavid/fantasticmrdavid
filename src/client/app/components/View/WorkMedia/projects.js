export default [
  {
    title: "SitePoint",
    tagline: "Front-End Engineer",
    target: "sitepoint",
    images: {
      tileOff: "/public/assets/images/projects/sitepoint_off.jpg",
      tileOn: "/public/assets/images/projects/sitepoint_on.jpg",
    },
    url: "https://www.sitepoint.com",
    products: "Company media website, Premium platform, forums",
    technologies: "HTML, CSS, ES6, React, Redux, Node.js, MobX, RoR, PHP",
    description: "As Front-End engineer I worked within an Agile development team to maintain and upgrade the company Articles WordPress and Premium RoR platforms. The revised articles site featured a new homepage and template designs with server-side rendered front-end components. We completely renovated the Premium members platform library design as well as shifted front-end components from Rails to React and Redux.",
    media: [
      {
        type: "website",
        platform: "mobile",
        thumbnail: "/public/assets/images/projects/sitepoint/sitepoint_mobile_preview.jpg",
        url: "https://www.sitepoint.com",
        title: "SitePoint mobile site",
      },
      {
        type: "website",
        platform: "desktop",
        thumbnail: "/public/assets/images/projects/sitepoint/sitepoint_preview.jpg",
        url: "https://www.sitepoint.com",
        title: "SitePoint desktop site",
      },
      {
        type: "website",
        platform: "desktop",
        thumbnail: "/public/assets/images/projects/sitepoint/sitepoint_premium_preview.jpg",
        url: "https://www.sitepoint.com/premium",
        title: "SitePoint Premium desktop site",
      },
      {
        type: "website",
        platform: "mobile",
        thumbnail: "/public/assets/images/projects/sitepoint/sitepoint_premium_mobile_preview.jpg",
        url: "https://www.sitepoint.com/premium",
        title: "SitePoint Premium mobile site",
      },
    ],
  },
  {
    title: "LEGO Flinders St",
    tagline: "CAD Art Project",
    target: "legoFlinders",
    images: {
      tileOff: "/public/assets/images/projects/legoFlinders_off.jpg",
      tileOn: "/public/assets/images/projects/legoFlinders_on.jpg",
    },
    products: "Artwork, prints",
    technologies: "LEGO Digital Designer, Photoshop, POV-Ray",
    description: "The combination of visiting Brickvention, unfulfilled childhood LEGO dreams, and a love for the great city that is Melbourne sparked off this project. The LDD model took a year to complete, over 43,000 parts, and many visits to the actual location. It was then converted and rendered in POV-Ray before adding final Melbournian touch ups in Photoshop.",
    media: [
      {
        type: "image",
        thumbnail: "/public/assets/images/projects/legoFlinders/legoFlinders_preview.jpg",
        url: "/public/assets/images/projects/legoFlinders/legoFlinders_full.jpg",
        orientation: "landscape",
        title: "LEGO Flinders St art print",
      },
    ],
  },
  {
    title: "General Assembly",
    tagline: "Lead Instructor",
    target: "ga",
    images: {
      tileOff: "/public/assets/images/projects/ga_off.jpg",
      tileOn: "/public/assets/images/projects/ga_on.jpg",
    },
    url: "https://generalassemb.ly",
    technologies: "HTML, CSS, JavaScript, PHP",
    description: "I currently teach the Front End Web Development (FEWD) part time course at General Assembly's Melbourne campus. Throughout the 10 week course, students learn HTML, CSS, JavaScript, and fundamental programming skills to build their own website project. It has been immensely rewarding with students moving on to roles ranging from Junior Developer to Project Manager.",
  },
  {
    title: "Tramsformation",
    tagline: "3D Printing & Design",
    target: "tramsformation",
    images: {
      tileOff: "/public/assets/images/projects/tramsformation_off.jpg",
      tileOn: "/public/assets/images/projects/tramsformation_on.jpg",
    },
    url: "http://tramsformer.com.au",
    products: "3D Models, artwork, website, transformable toy",
    technologies: "HTML, CSS, jQuery, PHP, MySQL, Photoshop, 3DS Max, 3D printing",
    description: "A personal project that started with a failed art competition entry. Featuring a website with shop, 3D models, T-shirts, prints and a 3D printed transformable figure. I have so much respect for the engineering in those old 80's toys.",
    media: [
      {
        type: "website",
        platform: "mobile",
        thumbnail: "/public/assets/images/projects/tramsformation/tramsformation_mobile_preview.jpg",
        url: "http://tramsformer.com.au",
        title: "Tramsformation mobile site",
      },
      {
        type: "website",
        platform: "desktop",
        thumbnail: "/public/assets/images/projects/tramsformation/tramsformation_desktop_preview.jpg",
        url: "http://tramsformer.com.au",
        title: "Tramsformation desktop site",
      },
      {
        type: "image",
        thumbnail: "/public/assets/images/projects/tramsformation/tramsformation_print_preview.jpg",
        url: "/public/assets/images/projects/tramsformation/tramsformation_print_full.jpg",
        orientation: "portrait",
        title: "Tramsformer art print",
      },
      {
        type: "image",
        thumbnail: "/public/assets/images/projects/tramsformation/tramsformation_transform_preview.jpg",
        url: "/public/assets/images/projects/tramsformation/tramsformation_transform_full.jpg",
        orientation: "portrait",
        title: "Tramsformer transfomration",
      },
      {
        type: "image",
        thumbnail: "/public/assets/images/projects/tramsformation/tramsformation_model_preview.jpg",
        url: "/public/assets/images/projects/tramsformation/tramsformation_model_full.jpg",
        orientation: "portrait",
        title: "Tramsformer model",
      },
    ],
  },
];
