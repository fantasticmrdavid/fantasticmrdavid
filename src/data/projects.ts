export type Project = {
  title: string,
  tagline: string,
  target: string,
  images: {
    [key: string]: string
    tileOn: string,
    tileOff: string,
  },
  url?: string,
  products?: string,
  technologies: string,
  description: string,
  media?: any[],
}

export default [
  {
    title: 'KindyNow',
    tagline: 'Lead Front-End Engineer',
    target: 'kindynow',
    images: {
      tileOff: '/assets/images/projects/kindynow_off.jpg',
      tileOn: '/assets/images/projects/kindynow_on.jpg',
    },
    url: 'https://www.kindynow.com',
    products: 'Child care search platform, Centre management platform, Mobile app, Admin platform',
    technologies: 'HTML, CSS, ES6, SSR React, React-Native, Redux, TypeScript, Node.js, Jest, Enzyme, AWS, CircleCI',
    description: 'KindyNow provides Australia\'s leading app and web service for helping parents find daycare for their children. As Lead Front-End engineer I architected and developed the company\'s flagship web assets, including a new child care search platform (Childcare Near Me), a rebuild of their existing booking management app (KindyNow), and admin tools for internal management of centres and bookings.<br /><br />I also managed and trained junior engineers, collaborated with the marketing team to prioritise and build features to maximise conversion goals, and provided guidance on UX and product strategy.',
    media: [
      {
        type: 'website',
        platform: 'mobile',
        thumbnail: '/assets/images/projects/kindynow/cnm_mobile_preview.jpg',
        url: 'https://childcarenearme.com.au',
        title: 'Childcare Near Me mobile site',
        slug: 'cnm_mobile_preview',
      },
      {
        type: 'website',
        platform: 'desktop',
        thumbnail: '/assets/images/projects/kindynow/cnm_search_preview.jpg',
        url: 'https://childcarenearme.com.au',
        title: 'Childcare Near Me search',
        slug: 'cnm_search_preview',
      },
      {
        type: 'website',
        platform: 'mobile',
        thumbnail: '/assets/images/projects/kindynow/kn_app_preview.jpg',
        url: 'https://www.kindynow.com',
        title: 'KindyNow Mobile App',
        slug: 'kn_app_preview',
      },
      {
        type: 'website',
        platform: 'desktop',
        thumbnail: '/assets/images/projects/kindynow/cnm_desktop_preview.jpg',
        url: 'https://childcarenearme.com',
        title: 'Childcare Near Me desktop site',
        slug: 'cnm_desktop_preview',
      },
      {
        type: 'website',
        platform: 'desktop',
        thumbnail: '/assets/images/projects/kindynow/kn_desktop_preview.jpg',
        url: 'https://www.kindynow.com',
        title: 'KindyNow desktop site',
        slug: 'kn_desktop_preview',
      },
    ],
  },
  {
    title: 'SitePoint',
    tagline: 'Front-End Engineer',
    target: 'sitepoint',
    images: {
      tileOff: '/assets/images/projects/sitepoint_off.jpg',
      tileOn: '/assets/images/projects/sitepoint_on.jpg',
    },
    url: 'https://www.sitepoint.com',
    products: 'Company media website, Premium platform, forums',
    technologies: 'HTML, CSS, ES6, React, Redux, Node.js, MobX, RoR, PHP',
    description: 'SitePoint is a hub built by developers for developers with the goal of sharing knowledge and educating others on incredible Internet things. Spanning articles, courses, books, screencasts, and a huge forum community, they are a leading source for the latest news on all things web development.<br /><br />As Front-End engineer I worked within an Agile development team to maintain and upgrade the company Articles WordPress and Premium RoR platforms. The revised articles site featured a new homepage and template designs with server-side rendered front-end components. We completely renovated the Premium members platform library design as well as shifted front-end components from Rails to React and Redux.',
    media: [
      {
        type: 'website',
        platform: 'mobile',
        thumbnail: '/assets/images/projects/sitepoint/sitepoint_mobile_preview.jpg',
        url: 'https://www.sitepoint.com',
        title: 'SitePoint mobile site',
        slug: 'sitepoint_mobile_preview',
      },
      {
        type: 'website',
        platform: 'desktop',
        thumbnail: '/assets/images/projects/sitepoint/sitepoint_preview.jpg',
        url: 'https://www.sitepoint.com',
        title: 'SitePoint desktop site',
        slug: 'sitepoint_preview',
      },
      {
        type: 'website',
        platform: 'desktop',
        thumbnail: '/assets/images/projects/sitepoint/sitepoint_premium_preview.jpg',
        url: 'https://www.sitepoint.com/premium',
        title: 'SitePoint Premium desktop site',
        slug: 'sitepoint_premium_preview',
      },
      {
        type: 'website',
        platform: 'mobile',
        thumbnail: '/assets/images/projects/sitepoint/sitepoint_premium_mobile_preview.jpg',
        url: 'https://www.sitepoint.com/premium',
        title: 'SitePoint Premium mobile site',
        slug: 'sitepoint_premium_mobile_preview',
      },
    ],
  },
  {
    title: 'LEGO Flinders St',
    tagline: 'CAD Art Project',
    target: 'legoFlinders',
    images: {
      tileOff: '/assets/images/projects/legoFlinders_off.jpg',
      tileOn: '/assets/images/projects/legoFlinders_on.jpg',
    },
    products: 'Artwork, prints',
    technologies: 'LEGO Digital Designer, Photoshop, POV-Ray',
    description: 'The combination of visiting <a href="http://www.brickventures.org.au/brickvention" target="_blank">Brickvention</a>, unfulfilled childhood LEGO dreams, and a love for the great city that is Melbourne left a nagging itch that my creative brain couldn\'t ignore. The LDD model took a year to complete, over 43,000 parts, and many visits to the actual location. It was then converted and rendered in POV-Ray before adding final Melbournian touch ups in Photoshop.',
    media: [
      {
        type: 'image',
        thumbnail: '/assets/images/projects/legoFlinders/legoFlinders_preview.jpg',
        url: '/assets/images/projects/legoFlinders/legoFlinders_full.jpg',
        orientation: 'landscape',
        title: 'LEGO Flinders St art print',
        slug: 'legoFlinders',
      },
    ],
  },
  {
    title: 'General Assembly',
    tagline: 'Lead Instructor',
    target: 'ga',
    images: {
      tileOff: '/assets/images/projects/ga_off.jpg',
      tileOn: '/assets/images/projects/ga_on.jpg',
    },
    url: 'https://generalassemb.ly',
    technologies: 'HTML, CSS, JavaScript, PHP',
    description: 'Starting out as a co-working space, General Assembly has grown into a pioneer in education and career transformation, specializing in todayâ€™s most in-demand skills. They are a leading source for training, staffing, and career transitions, fostering a flourishing community of professionals pursuing careers they love.<br /><br />I taught the Front End Web Development (FEWD) part time course at General Assembly\'s Melbourne campus. Throughout the 10 week course, students learnt HTML, CSS, JavaScript, and fundamental programming skills to build their own website project. It was immensely rewarding with students moving on to roles ranging from Junior Developer to Project Manager.',
  },
  {
    title: 'Tramsformation',
    tagline: '3D Printing & Design',
    target: 'tramsformation',
    images: {
      tileOff: '/assets/images/projects/tramsformation_off.jpg',
      tileOn: '/assets/images/projects/tramsformation_on.jpg',
    },
    products: '3D Models, artwork, website, transformable toy',
    technologies: 'HTML, CSS, jQuery, PHP, MySQL, Photoshop, 3DS Max, 3D printing',
    description: 'A personal project that started with a failed art competition entry. Featuring a website with shop, 3D models, T-shirts, prints and a 3D printed transformable figure. I have so much respect for the engineering in those old 80\'s toys.',
    media: [
      {
        type: 'website',
        platform: 'mobile',
        thumbnail: '/assets/images/projects/tramsformation/tramsformation_mobile_preview.jpg',
        title: 'Tramsformation mobile site',
        slug: 'tramsformation_mobile',
      },
      {
        type: 'website',
        platform: 'desktop',
        thumbnail: '/assets/images/projects/tramsformation/tramsformation_desktop_preview.jpg',
        title: 'Tramsformation desktop site',
        slug: 'tramsformation_desktop',
      },
      {
        type: 'image',
        thumbnail: '/assets/images/projects/tramsformation/tramsformation_print_preview.jpg',
        url: '/assets/images/projects/tramsformation/tramsformation_print_full.jpg',
        orientation: 'portrait',
        title: 'Tramsformer art print',
        slug: 'tramsformation_print',
      },
      {
        type: 'image',
        thumbnail: '/assets/images/projects/tramsformation/tramsformation_transform_preview.jpg',
        url: '/assets/images/projects/tramsformation/tramsformation_transform_full.jpg',
        orientation: 'portrait',
        title: 'Tramsformer transformation',
        slug: 'tramsformation_transform',
      },
      {
        type: 'image',
        thumbnail: '/assets/images/projects/tramsformation/tramsformation_model_altmode_preview.jpg',
        url: '/assets/images/projects/tramsformation/tramsformation_model_altmode_full.jpg',
        orientation: 'landscape',
        title: 'Tramsformer model tram mode',
        slug: 'tramsformation_model_altmode',
      },
      {
        type: 'image',
        thumbnail: '/assets/images/projects/tramsformation/tramsformation_model_preview.jpg',
        url: '/assets/images/projects/tramsformation/tramsformation_model_full.jpg',
        orientation: 'portrait',
        title: 'Tramsformer model',
        slug: 'tramsformation_model',
      },
    ],
  },
];
