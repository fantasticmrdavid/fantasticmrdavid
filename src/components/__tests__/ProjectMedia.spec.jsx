import React from 'react';
import { shallow } from 'enzyme';
import Ui from 'components/ProjectMedia/ui';
import projects from 'data/projects';

projects.map(p => {
  if (p.media) {
    describe(`Component -> ProjectMedia -> ${p.title}`, () => {
      p.media.map(m => {
        it(`should render the ProjectMedia UI (src: "${m.thumbnail}") without throwing an error`, () => {
          expect(shallow(<Ui {...m} />)).toMatchSnapshot();
        });

        return m;
      });
    });
  }

  return p;
});
