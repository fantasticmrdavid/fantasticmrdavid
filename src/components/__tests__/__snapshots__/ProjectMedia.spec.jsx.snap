// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component -> ProjectMedia -> KindyNow should render the ProjectMedia UI (src: "/assets/images/projects/kindynow/cnm_desktop_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="desktop"
  slug="cnm_desktop_preview"
  thumbnail="/assets/images/projects/kindynow/cnm_desktop_preview.jpg"
  title="Childcare Near Me desktop site"
  type="website"
  url="https://childcarenearme.com"
/>
`;

exports[`Component -> ProjectMedia -> KindyNow should render the ProjectMedia UI (src: "/assets/images/projects/kindynow/cnm_mobile_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="mobile"
  slug="cnm_mobile_preview"
  thumbnail="/assets/images/projects/kindynow/cnm_mobile_preview.jpg"
  title="Childcare Near Me mobile site"
  type="website"
  url="https://childcarenearme.com.au"
/>
`;

exports[`Component -> ProjectMedia -> KindyNow should render the ProjectMedia UI (src: "/assets/images/projects/kindynow/cnm_search_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="desktop"
  slug="cnm_search_preview"
  thumbnail="/assets/images/projects/kindynow/cnm_search_preview.jpg"
  title="Childcare Near Me search"
  type="website"
  url="https://childcarenearme.com.au"
/>
`;

exports[`Component -> ProjectMedia -> KindyNow should render the ProjectMedia UI (src: "/assets/images/projects/kindynow/kn_app_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="mobile"
  slug="kn_app_preview"
  thumbnail="/assets/images/projects/kindynow/kn_app_preview.jpg"
  title="KindyNow Mobile App"
  type="website"
  url="https://www.kindynow.com"
/>
`;

exports[`Component -> ProjectMedia -> KindyNow should render the ProjectMedia UI (src: "/assets/images/projects/kindynow/kn_desktop_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="desktop"
  slug="kn_desktop_preview"
  thumbnail="/assets/images/projects/kindynow/kn_desktop_preview.jpg"
  title="KindyNow desktop site"
  type="website"
  url="https://www.kindynow.com"
/>
`;

exports[`Component -> ProjectMedia -> LEGO Flinders St should render the ProjectMedia UI (src: "/assets/images/projects/legoFlinders/legoFlinders_preview.jpg") without throwing an error 1`] = `
<Connect(Component)
  orientation="landscape"
  slug="legoFlinders"
  thumbnail="/assets/images/projects/legoFlinders/legoFlinders_preview.jpg"
  title="LEGO Flinders St art print"
  type="image"
  url="/assets/images/projects/legoFlinders/legoFlinders_full.jpg"
/>
`;

exports[`Component -> ProjectMedia -> SitePoint should render the ProjectMedia UI (src: "/assets/images/projects/sitepoint/sitepoint_mobile_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="mobile"
  slug="sitepoint_mobile_preview"
  thumbnail="/assets/images/projects/sitepoint/sitepoint_mobile_preview.jpg"
  title="SitePoint mobile site"
  type="website"
  url="https://www.sitepoint.com"
/>
`;

exports[`Component -> ProjectMedia -> SitePoint should render the ProjectMedia UI (src: "/assets/images/projects/sitepoint/sitepoint_premium_mobile_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="mobile"
  slug="sitepoint_premium_mobile_preview"
  thumbnail="/assets/images/projects/sitepoint/sitepoint_premium_mobile_preview.jpg"
  title="SitePoint Premium mobile site"
  type="website"
  url="https://www.sitepoint.com/premium"
/>
`;

exports[`Component -> ProjectMedia -> SitePoint should render the ProjectMedia UI (src: "/assets/images/projects/sitepoint/sitepoint_premium_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="desktop"
  slug="sitepoint_premium_preview"
  thumbnail="/assets/images/projects/sitepoint/sitepoint_premium_preview.jpg"
  title="SitePoint Premium desktop site"
  type="website"
  url="https://www.sitepoint.com/premium"
/>
`;

exports[`Component -> ProjectMedia -> SitePoint should render the ProjectMedia UI (src: "/assets/images/projects/sitepoint/sitepoint_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="desktop"
  slug="sitepoint_preview"
  thumbnail="/assets/images/projects/sitepoint/sitepoint_preview.jpg"
  title="SitePoint desktop site"
  type="website"
  url="https://www.sitepoint.com"
/>
`;

exports[`Component -> ProjectMedia -> Tramsformation should render the ProjectMedia UI (src: "/assets/images/projects/tramsformation/tramsformation_desktop_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="desktop"
  slug="tramsformation_desktop"
  thumbnail="/assets/images/projects/tramsformation/tramsformation_desktop_preview.jpg"
  title="Tramsformation desktop site"
  type="website"
/>
`;

exports[`Component -> ProjectMedia -> Tramsformation should render the ProjectMedia UI (src: "/assets/images/projects/tramsformation/tramsformation_mobile_preview.jpg") without throwing an error 1`] = `
<Memo()
  platform="mobile"
  slug="tramsformation_mobile"
  thumbnail="/assets/images/projects/tramsformation/tramsformation_mobile_preview.jpg"
  title="Tramsformation mobile site"
  type="website"
/>
`;

exports[`Component -> ProjectMedia -> Tramsformation should render the ProjectMedia UI (src: "/assets/images/projects/tramsformation/tramsformation_model_altmode_preview.jpg") without throwing an error 1`] = `
<Connect(Component)
  orientation="landscape"
  slug="tramsformation_model_altmode"
  thumbnail="/assets/images/projects/tramsformation/tramsformation_model_altmode_preview.jpg"
  title="Tramsformer model tram mode"
  type="image"
  url="/assets/images/projects/tramsformation/tramsformation_model_altmode_full.jpg"
/>
`;

exports[`Component -> ProjectMedia -> Tramsformation should render the ProjectMedia UI (src: "/assets/images/projects/tramsformation/tramsformation_model_preview.jpg") without throwing an error 1`] = `
<Connect(Component)
  orientation="portrait"
  slug="tramsformation_model"
  thumbnail="/assets/images/projects/tramsformation/tramsformation_model_preview.jpg"
  title="Tramsformer model"
  type="image"
  url="/assets/images/projects/tramsformation/tramsformation_model_full.jpg"
/>
`;

exports[`Component -> ProjectMedia -> Tramsformation should render the ProjectMedia UI (src: "/assets/images/projects/tramsformation/tramsformation_print_preview.jpg") without throwing an error 1`] = `
<Connect(Component)
  orientation="portrait"
  slug="tramsformation_print"
  thumbnail="/assets/images/projects/tramsformation/tramsformation_print_preview.jpg"
  title="Tramsformer art print"
  type="image"
  url="/assets/images/projects/tramsformation/tramsformation_print_full.jpg"
/>
`;

exports[`Component -> ProjectMedia -> Tramsformation should render the ProjectMedia UI (src: "/assets/images/projects/tramsformation/tramsformation_transform_preview.jpg") without throwing an error 1`] = `
<Connect(Component)
  orientation="portrait"
  slug="tramsformation_transform"
  thumbnail="/assets/images/projects/tramsformation/tramsformation_transform_preview.jpg"
  title="Tramsformer transformation"
  type="image"
  url="/assets/images/projects/tramsformation/tramsformation_transform_full.jpg"
/>
`;

exports[`Component -> ProjectMedia -> Tramsformation should render the ProjectMedia UI (src: "/assets/images/projects/tramsformation/tramsformer_w_1_preview.jpg") without throwing an error 1`] = `
<Connect(Component)
  orientation="landscape"
  slug="tramsformation_w_1"
  thumbnail="/assets/images/projects/tramsformation/tramsformer_w_1_preview.jpg"
  title="Tramsformer W-Class Robot Mode"
  type="image"
  url="/assets/images/projects/tramsformation/tramsformer_w_1_full.jpg"
/>
`;

exports[`Component -> ProjectMedia -> Tramsformation should render the ProjectMedia UI (src: "/assets/images/projects/tramsformation/tramsformer_w_2_preview.jpg") without throwing an error 1`] = `
<Connect(Component)
  orientation="landscape"
  slug="tramsformation_w_2"
  thumbnail="/assets/images/projects/tramsformation/tramsformer_w_2_preview.jpg"
  title="Tramsformer W-Class Tram Mode"
  type="image"
  url="/assets/images/projects/tramsformation/tramsformer_w_2_full.jpg"
/>
`;
